rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/LicenseController.java

```diff
@@ -562,6 +562,19 @@ public ResponseEntity deleteLicenseType(
         User sw360User = restControllerHelper.getSw360UserFromAuthentication();
         RequestStatus status = licenseService.deleteLicenseType(id, sw360User);
 
-        return ResponseEntity.ok("License type deleted successfully: " + status.name());
+        Map<String, String> response = new HashMap<>();
+        response.put("status", status.name());
+
+        switch (status) {
+        case SUCCESS:
+            response.put("message", "License type deleted successfully.");
+            return ResponseEntity.ok(response);
+        case IN_USE:
+            response.put("message", "Cannot delete license type because it is currently in use.");
+            return ResponseEntity.status(HttpStatus.CONFLICT).body(response);
+        default:
+            response.put("message", "Failed to delete license type due to unknown reason.");
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(response);
```

Comment:
Please add a case for `RequestStatus.ACCESS_DENIED` and throw `AccessDeniedException`